### DISCUSSION TITLE
New Discourse forum

### DISCUSSION THREAD

**Discussion by hecrj** [hecrj/iced#2052] [General]
| :warning: WARNING           |
|:----------------------------|
| As of today, this post is outdated. Our discussion forum is Zulip. |
| :arrow_right: https://iced.zulipchat.com/ |

---------------------------------------------------
I have set up a Discourse forum for iced. You can access it here: https://discourse.iced.rs/

# Why? Didn't we already have Discord?
Discord encourages low-effort, synchronous communication. As the community grows, this kind of communication becomes very inefficient.

Attention is a limited resource and a lot of low-effort, synchronous communication requires continuous attention. As the server grows, it's a struggle for me and the core team to keep up with the community in a focused, meaningful way.

Additionally, when low effort posts have the same visibility as high effort ones, something is wrong. Our community is not encouraging the kind of work we want to see.

Discourse is a forum that encourages high-effort, asynchronous communication. It is also open-source and public by default.

# What will happen?
For now, we are just drive-testing.

Eventually, we will disable GitHub discussions and start sending users to the Discourse forum. The #questions, #discussions, and #showcase channels on the Discord server will be moved there as well.

The Discord server will become a place for hanging out and chatting with iced users, asking quick questions, or just socializing. It will be mainly community-driven.

The Discourse forum will be the place to ask more fleshed out questions, share ideas, and overall get involved in more focused work. The core team will mainly be present there.

**Comment by lufte** [comment]
I like this. Will all existing github discussion be moved to discourse somehow? There's valuable knowledge in here.

Edit: I'm glad to help with a migration if there's intention to do it.

**Reply by hecrj** [reply]
No plans to migrate anything for now. All GitHub discussions will stay accessible in read-only mode.

Furthermore, the library changes quite heavily between releases, so I don't believe we would be losing that much.

**Comment by inferiorhumanorgans** [comment]
Is the forum dead?

```
$ nslookup
> server 1.1.1.1
Default server: 1.1.1.1
Address: 1.1.1.1#53
> discourse.iced.rs
Server:         1.1.1.1
Address:        1.1.1.1#53

** server can't find discourse.iced.rs: NXDOMAIN
> server 8.8.8.8
Default server: 8.8.8.8
Address: 8.8.8.8#53
> discourse.iced.rs
Server:         8.8.8.8
Address:        8.8.8.8#53

** server can't find discourse.iced.rs: NXDOMAIN
>
```

**Reply by pml68** [reply]
Yup, Discourse is out and Zulip's in at https://iced.zulipchat.com
